pages/common/poetry-export.md

```diff
@@ -0,0 +1,29 @@
+# poetry export
+
+> Export Poetry's lock file to other formats.
+> Provided by the Export Poetry Plugin.
+> More information: <https://github.com/python-poetry/poetry-plugin-export#usage>.
+
+- Export dependencies to a `requirements.txt` file:
+
+`poetry export {{[-o|--output]}} {{requirements.txt}}`
+
+- Export dependencies including development dependencies:
+
+`poetry export --dev {{[-o|--output]}} {{requirements-dev.txt}}`
```

Comment:
```suggestion
`poetry export {{[-o|--output]}} {{requirements-dev.txt}} --dev`
```
Align the same option on all examples so that it's easier to read.